src/services/nodegit/NodeGitService.js

```diff
@@ -0,0 +1,275 @@
+const Git = require("nodegit");
+const path = require("path");
+const _ = require('lodash');
+
+/**
+ * @requires Models
+ */
+import {
+    PlaybookModel
+} from '../../models/playbook/index';
+
+/**
+ * @requires Services 
+ */
+const FilesService = require('../utils/FilesService');
+const DiffService = require('../diff/DiffService');
+
+class NodeGitService
+{
+    async createBlueprintsFolder(githubUrl)
+    {
+        // -- This is temporary to ensure we have a directory to clone to. We should resort to /tmp later
+        FilesService.createFolder(path.resolve(__dirname, '../../../../'), "git-projects");
```

Comment:
added Todo in gitlab project board https://gitlab.com/master-class.io/microclients/client_playbook_sdk/-/issues/12